doped/pycdt/utils/parse_calculations.py

```diff
@@ -477,6 +480,7 @@ def from_paths(
         except RuntimeError as exc:
             # try transformation.json
             # if auto site-matching failed, try use transformation.json
+            # Bonan: I don't think does anything?
```

Comment:
This tries to use the `transformation.json` file, if present, to do the site matching, if the default `doped` routines fail

Reply:
Right I see, so all it does is trying to find `defect_site_idx`, the code path is rather confusing